### PR TITLE
Docs GitHub contributions

### PR DESCRIPTION
**Pull request by lauriemerrell**
# Overall Description

As the data services team has grown, we thought it would be prudent to put together some "official" GitHub contribution guidelines. Once this is added to the docs site, it can be linked from `CONTRIBUTING.md` files in all repos that want to opt in to these norms. 

This draft is based heavily on Jarvus's internal GitHub guide, but with some updates to make it applicable to Cal-ITP.

## Checklist for all PRs

- [x] Run `pre-commit run --all-files` to make sure markdown/lint passes
- [x] ~Link this pull request to all issues that it will close using keywords (see GitHub docs about [Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). Also mention any issues that are partially addressed or are related.~

## Docs changes checklist

- [x] Include this section whenever any change in the `docs` folder occurs, otherwise please omit this section.
- [x] **If you haven't already, review the [Contribute to the Docs](https://docs.calitp.org/data-infra/contribute/overview.html) section of the data services documentation for best practices, common formatting, and more**
- [x] Make sure the preview website was able to be generated
- [x] Fill out the following section describing what docs were added/updated

This PR creates `github/contributing_to_repos.md` in order to document GitHub contribution guidelines.

### CODE REVIEW

**Review by holly-g — APPROVED** [review]
Thanks for providing clarity on our workflow process!

**Comment by lauriemerrell** [comment]
(Awaiting further review before merging!)

File: docs/github/contribute_to_repos.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Move your PR out of draft status when it’s ready for review and merging.
+* Maintain responsibility for all PRs you open until they are merged, keeping an eye out for comments and merging promptly after approval is received.
+* PR authors should merge PRs once approval is received.
+
+
+## Code Reviews - Reviewer:
+* Reviewers should not submit comments on PRs that are still in draft status unless specifically requested by the PR author.
+* Practice [conventional comments](https://conventionalcomments.org/) to make it clear to recipients of review what notes require what action, and what’s just feedback.
+* If you are approving a PR and leave a comment with your approval, the PR author should understand that that comment is **optional** to address.
+   * If you have a change that you think should be **required** before the change is merged, please request changes on the PR rather than approving.
```
Comment (atvaccaro) [inline docs/github/contribute_to_repos.md:49]: google has some decent advice on how to leave review comments https://google.github.io/eng-practices/review/reviewer/

**Review by atvaccaro — APPROVED** [review]
:shipit:

File: docs/github/contribute_to_repos.md (hunk 1 above)
Comment (holly-g) [reply docs/github/contribute_to_repos.md:49]: **thought (belated):** I like Google's `eng-practices` doc, particulary the section on 'What to look for in a code review'. James and Mjumbe have been following https://conventionalcomments.org/ when leaving PR comments. I mistakenly thought this was linked in the README under `Contributing`, but only [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) is listed. 

**suggestion (non-blocking):** Could we direct folks to Google's [How to do a code review](https://google.github.io/eng-practices/review/reviewer/) recommendations for their reference, and link https://conventionalcomments.org/ for leaving actionable comments in PRs? (_I am also pretty new to this format -- I hope I'm doing this right!)_

File: docs/github/contribute_to_repos.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+These guidelines are meant to provide a foundation for collaboration in Cal-ITP's data services repos (primarily [#data-infra](https://github.com/cal-itp/data-infra).)
+
+## Issues:
+* When submitting an issue, please make it clear what the acceptance criteria are; i.e., what is sufficient to consider a fix “done” for the issue.
+    * For example: The following issue is vague and could fall victim to an unhelpful cycle of swirl and scope creep:
+      > Add best practices to reports
+
+    * The following issue is clearer. If scope were to increase (for example, the author realizes that they also want a chart), it would be clearer that that should be created as a new, follow-up request.
+       > Add a new section to reports called ‘Best Practices Violations’. It should be inserted immediately after and appear just like the existing critical validation errors table and include validation notices W1, W2, and W3.
```
Comment (evansiroky) [inline docs/github/contribute_to_repos.md:11]: I support this as a good framework for creating focused GitHub issues, however, I think it'd be a good idea to also note the proper venue for proposing larger changes that need to be scoped out.

**Review by evansiroky — CHANGES_REQUESTED** [review]
I like everything in here, but do think it'd be a good idea to point people to a certain place when they want to make an issue that needs further scoping. For example, in this use case, the author may not initially know what acceptance criteria they want for the best practices yet, but do definitely know that they want the best practices and don't want this item to get lost. In that case, this doc should give suggestions for where that author should make their fuzzy request.

File: docs/github/contribute_to_repos.md (hunk 2 above)
Comment (lauriemerrell) [reply docs/github/contribute_to_repos.md]: Added in https://github.com/cal-itp/data-infra/pull/1288/commits/920ee0ba31aa713971e1e8ee8b85ca85c8206819 and clarified purpose of this expectation in https://github.com/cal-itp/data-infra/pull/1288/commits/213c4b5ec4c5bf61370bf405b2af9ee3cd482cb4. Please let me know if additional context or clarification is needed.

Comment (evansiroky) [reply docs/github/contribute_to_repos.md]: Works for me

**Review by evansiroky — APPROVED** [review]

**Pull request merged**